content/docs/user-guide/visualizing-plots.md

```diff
@@ -0,0 +1,385 @@
+# Visualizing Plots
+
+A typical workflow for DVC plots is:
+
+1. Save data to a [supported file format](#supported-file-formats).
+
+```csv
+fpr,tpr,threshold
+0.0,0.0,1.5
+1.0,1.0,0.0
+```
+
+2. [Define plots](#defining-plots), optionally using
+   [templates](#plot-templates-data-series-only) to configure how to visualize
+   the data.
+
+```yaml
+plots:
+  auc.tsv:
+    x: fpr
+    y: tpr
+```
+
+3. [Show](/doc/command-reference/plots/show) all plots in a single view or
+   report.
```

Comment:
```suggestion
3. `dvc plots show` all plots in a single view or report.
```

Reply:
Keeping this one as is. Don't want to introduce commands here since it's too low level and inconsistent with the others, plus it's inaccurate for showing plots in VS Code and Studio.